.github/workflows/unversioned_pages.yml

```diff
@@ -1,4 +1,6 @@
 name: Copy unversioned pages
+# Unversioned pages are pages with general content that shouldn't change from
+# version to version. Examples: community, developers, naps, release, roadmaps
```

Comment:
I think we should probably link to https://napari.org/stable/developers/contributing/documentation/docs_deployment.html#documentation-and-website-deployment